app/main.py

```diff
@@ -22,7 +25,10 @@ def load_config(self) -> None:
             'top_up_threshold_gb', 'top_up_gb', 'max_gb'
         ]:
             if self.config.get(attr) is None:
-                raise Exception(f'Config file must specify "{attr}" attribute.')
+                e = os.environ.get(f'lifeguard_{attr}'.upper())
+                if e is None:
+                    raise Exception(f'Config file or env var must specify "{attr}" attribute.')
+                setattr(self, attr, e)
             setattr(self, attr, self.config[attr])
```

Comment:
This will raise an exception if no config file or overwrite the env var if there is. Maybe restructure this to something like:

    val = self.config.get(attr)
    if val is None:
        val = os.environ.get(f'lifeguard_{attr}'.upper())
    if val is None:
        raise Exception(f'Config file or env var must specify "{attr}" attribute.')
    setattr(self, attr, val)